src/verify.py

```diff
@@ -0,0 +1,14 @@
+# Write a software that verifies if a number is present in a pre-defined array.
+# Output example:
+# Insert number 3
+# The number 3 is [not] present in the array.
+
+if __name__ == '__main__':
+  N = [3, 4, 5, 1, 2, 3, 4, 9, 13, 0]
+
+  value = input('Insert number: ')
+
+  if ( int( value ) in N ): 
```

Comment:
Il cast da 'str' a 'int' può creare un'eccezione. Potresti assicurarti che tutto fili liscio mettendo in 'and' la condizione

```python
if value.isnumeric() and int(n) in N:
    ...
```